### ISSUE TITLE
docker_container host_entry --> host

### ISSUE THREAD

**Issue by milligana** [hashicorp/terraform#9350]
Somebody needs to update the docker container help markdown to show that the "Extra Hosts" 'host_entry' tag actually is 'host'.  This is certainly on 0.7.5, but probably earlier versions too.

**Comment by dmportella** [comment]
I am about to drop in a pr I can make this change

**Comment by dmportella** [comment]
Yeah the code base is now different then the docs even the types described in the documentationa re round they are not ints they are strings

```
"host": &schema.Schema{
	Type:     schema.TypeSet,
	Optional: true,
	ForceNew: true,
	Elem: &schema.Resource{
		Schema: map[string]*schema.Schema{
			"ip": &schema.Schema{
				Type:     schema.TypeString,
				Optional: true,
				ForceNew: true,
			},

			"host": &schema.Schema{
				Type:     schema.TypeString,
				Optional: true,
				ForceNew: true,
			},
		},
	},
	Set: resourceDockerHostsHash,
},
```

**Comment by dmportella** [comment]
PR sent

**Comment by stack72** [comment]
Closed via #9367
